### ISSUE TITLE
Support for SpecFlow assembly references (non-NuGet)

### ISSUE THREAD

**Issue by gasparnagy** [specsolutions/deveroom-visualstudio#14]
Currently Deveroom detects the installed SpecFlow version from the NuGet package references, therefore it is not able to detect SpecFlow if you reference the SpecFlow assemblies as assembly references.

If Deveroom cannot detect SpecFlow, certain features, like Navigation and coloring undefined steps will not work.

Later we plan to support using SpecFlow assembly references as well, you can use this issue to track the progress.

**Comment by LadaVarga** [comment]
Hello, do you have aby ETA fór this? Do you have aby branch where i can look or place in the code where i can start?

**Comment by gasparnagy** [comment]
Thx for offering help.
I will hopefully be able to make the GitHub source code publish by tomorrow. This is one of the top priority things to solve after that. I have some design ideas, I will share them with you and you can see if you can contribute.

**Comment by AnnieOletubo** [comment]
Hi All,
Have you any update on that point?

**Comment by gasparnagy** [comment]
Not yet, but scheduled for the next release. 

@AnnieOletubo @LadaVarga would you be available for some beta testing once I have something to try out?

**Comment by AnnieOletubo** [comment]
Of course I can.
Thanks a lot

**Comment by gasparnagy** [comment]
@AnnieOletubo there is a preview version available in PR #43. 

It detects some bin-loaded SpecFlow versions automatically, but for code-behind generation, you need to add a "deveroom.json" file to the project root (there is an item in the Deveroom section of the "Add new item" dialog). You will need to set:

```
{
  ...
  "specFlow": {
    "isSpecFlowProject": true,
    "generatorFolder": "your-specflow-path\\tools"
  }
}
```

https://dev.azure.com/specsolutions/_apis/resources/Containers/3728208?itemPath=vsix%2FDeveroom.VisualStudio.Package.vsix

You need to manually uninstall your extension first, then double click on the downloaded vsix. Once you are done with the test, uninstall the preview and install Deveroom again from the extension manager. 

//cc @LadaVarga

**Comment by gasparnagy** [comment]
Will be released hopefully today. See documentation at http://speclink.me/deveroomsfassref

**Comment by gasparnagy** [comment]
Released as v1.4.0

**Comment by AnnieOletubo** [comment]
Sorry I was on hollidays so I didnt see your message.

Thanks for the update . I run to upgrade my version!
Thanks :) 👍
